navigation/screens/review-form.tsx

```diff
@@ -11,14 +11,15 @@ import { createReview } from "@/controllers/reviewController";
 import Main from "./home";
 import { useNavigation } from "@react-navigation/native";
 
-export default function TabReviewForm() {
+export default function TabReviewForm(
+  username: string,
+  bathroomID: string,
+  time: Date,
```

Comment:
time does not need to be passed into the constructor. we will call a library inside this component to get the current time. please remove this line